Source/Project64/User Interface/Frame Per Second Class.cpp

```diff
@@ -58,7 +58,7 @@ void CFramePerSecond::Reset (bool ClearDisplay)
 
 void CFramePerSecond::UpdateViCounter ( void )
 {
-	if (m_iFrameRateType != FR_VIs && m_iFrameRateType != FR_PERCENT)
+	if (m_iFrameRateType != FR_VIs && m_iFrameRateType != FR_PERCENT && m_iFrameRateType != FR_VIs_DLs)

```

Comment:
this might be right .. but I am not sure it looks like it could be wrong at first glance, I need to work out exactly if it is wrong or not